tests/hashing/subtests/avoid-cascade/rollup/lib/import-map-plugin.js

```diff
@@ -65,10 +65,13 @@ export default function importMapPlugin() {
         });
       }
 
+      const hash = createHash('md5');
+      hash.update(JSON.stringify(importMap));
```

Comment:
I don't think this is necessary, but I'll work on that.

Reply:
It's not necessary, but it would leave the import-map as not-cacheable. As they can get quite large this leave you with an large uncacheable import-map resp. HTML file. I don't think that's intended.

Reply:
I guess the fully consistent way would be for me to bundle systemjs & the map together in a bundle.